sync/fromDisk/questions.js

```diff
@@ -28,7 +28,9 @@ function getParamsForQuestion(q) {
         options: q.options,
         client_files: q.clientFiles || [],
         topic: q.topic,
-        grading_method: q.gradingMethod || 'Internal',
+        grading_method_internal: !q.gradingMethod || q.gradingMethod === 'Internal',
```

Comment:
Here's where we'll change config parsing to allow multiple "grading methods" per assignment.

Reply:
So it seems that the idea is to still configure the question, and then grade the submission on the basis of the flags in the configuration file. This looks good!

We then create a grading job and insert some of the internal, external, and manual boolean fields that are updated. I am starting to see how you are getting pulled into the situation where this information lives on the submission and grading_job tables. I think we may want to join this information though. It will require modifying some queries. @mwest1066

Reply:
Yup, configure the question, then create discrete jobs for each type of grading method that we'll deal with. We'd have to look at the `gj.grading_method_manual` field to insert an initial job, and then we should be able to just look for any incomplete grading job in the DB. I know this gets hairy with how we handle conflicts in the spin lock prs (since we also have manual jobs for pending grading info... stuff), but it aligns with how we discover and handle external grading jobs.